inginious/frontend/plugins/upcoming_tasks/templates/upcoming_task_list.html

```diff
@@ -0,0 +1,53 @@
+{# This file is part of INGInious. See the LICENSE and the COPYRIGHTS files for #}
+{# more information about the licensing of this file. #}
+
+{% set registered = user_manager.course_is_user_registered(course) %}
+
+<div id="course" class="section tasks_list card mb-4">
+
+    <div class="content list-group list-group-flush tasks-list">
+        {% for task in tasks %}
+            {% set taskid = task.get_id() %}
+            {% if taskid in tasks_data %}
+                {% set succeeded = tasks_data[taskid]["succeeded"] %}
+                {% set completion = tasks_data[taskid]["grade"] %}
+                <a href="{{ get_homepath() }}/course/{{course.get_id()}}/{{taskid}}" class="list-group-item list-group-item-action
+                    {% if not task.get_accessible_time().is_open() %} disabled {% endif %}">
+                    <div class="row">
+                        <div class="col-xs-12 col-md-7 pl-1">
+                            <i style="color: {% if succeeded %}#468847{% else %}transparent{% endif %};" class="fa fa-check"></i>
+                            {{ task.get_name(user_manager.session_language()) }}
+                            {% if not task.get_accessible_time().is_open() %}
+                                - <b> {{ _("deadline reached") }} </b>
+                            {% endif %}
+                        </div>
+                        <div class="col-xs-12 col-md-5">
+                            {% if registered %}
+                                <div class="progress">
+                                    <div class="progress-bar bg-success" aria-valuenow="{{ completion | int }}" aria-valuemin="0" aria-valuemax="100" style="width: {{ completion | int }}%">
+                                        {% if completion.is_integer() %}{{ completion | int }}{% else %}{{ completion }}{% endif %} %
+                                    </div>
+                                </div>
+                            {% endif %}
+							
+                        </div>
+                    </div>
+					
+					<div class="col-xs-12 col-md-5">
```

Comment:
There tabs instead of spaces here. I'm also wondering why this block alone outside of a ``row`` block. I think the content of this block should be put in the block at line 17 : https://github.com/UCL-INGI/INGInious/pull/655/files#diff-48e9489117a9d34504271681302813d0e5d50ea01158dc6a9209054e2e7da72eR17

Reply:
Well sounds obvious I'm not very used to html haha :D
I've put the block in the previous row block as you advised.
I can't see any tabs from my pycharm interface... ?
It's looking like this on the page now: